graylog2-web-interface/src/components/search/MessageFieldDescription.jsx

```diff
@@ -50,6 +50,15 @@ class MessageFieldDescription extends React.Component {
     SearchStore.addSearchTerm(this.props.fieldName, this.props.fieldValue);
   };
 
+  _getFieldValue() {
+    if (this.props.isDecorated && ('type' in this.props.fieldValue) && (this.props.fieldValue.type === 'a')) {
+      const link = this.props.fieldValue.href;
+      return React.createElement('a', {href: link}, link);
```

Comment:
Please add spaces between the object braces: https://eslint.org/docs/rules/object-curly-spacing